### PR TITLE
[Android] Updated document for nnstreamer multi-model app v2

### PR DESCRIPTION
**Pull request by again4you**
This PR is a succeeding PR of #2359. 
* Apply original patch of @leemgs
* Update some guides and fix the typoes
* Apply all comments of reviewers

**Self evaluation:**
1. Build test: [ ]Passed [ ]Failed [*]Skipped
2. Run test: [ ]Passed [ ]Failed [*]Skipped

### CODE REVIEW

File: api/android/README.md [hunk 1]
```diff
@@ -151,10 +160,11 @@ $ bash ./api/android/build-android-lib.sh
 
 After building the Android API, you can find the library(.aar) in `$NNSTREAMER_ROOT/android_lib`.
 - Build result
-  1. nnstreamer.aar: NNStreamer library
-  2. nnstreamer-native.zip: shared objects and header files for native developer
+  1. nnstreamer-[BUILD_DATE].aar: NNStreamer library for Android
+  2. nnstreamer-native-[BUILD_DATE].zip: shared objects and header files for native developer
+  3. nnstreamer-static-libs-[BUILD_DATE].zip: static library files (i.e. .a)
```
Comment (jaeyun-jung) [inline api/android/README.md:165]: Actually static libs are unnecessary for app developers (just for the license)
and I updated mk files to build shared library now.

Comment (again4you) [reply api/android/README.md:165]: I agreed.

File: api/android/README.md [hunk 2]
```diff
@@ -5,36 +5,36 @@
...
 #
 export JAVA_HOME=/opt/android-studio/jre            # JRE path in Android Studio
-export ANDROID_DEV_ROOT=$HOME/android               # Set your own path (The default path will be "$HOME/Android".)
-#
-# $ANDROID_DEV_ROOT/tools/sdk                    # Android SDK root directory (default location: $HOME/Android/Sdk)
-# $ANDROID_DEV_ROOT/tools/ndk                    # Android NDK root directory (default location: $HOME/Android/Sdk/ndk/<ndk-version>)
-# $ANDROID_DEV_ROOT/gstreamer-1.0                # GStreamer binaries
-# $ANDROID_DEV_ROOT/workspace/nnstreamer         # NNStreamer cloned git repository
-#
+export ANDROID_DEV_ROOT=$HOME/android               # Set your own path (default location: "$HOME/Android".)
```
Comment (leemgs) [inline api/android/README.md:31]: @again4you , If we try to use '**android**', NNStreamer users will be often confused due to similar word (e.g. **android** , **Android** [default]).  So, I want  you to use "**Android** [default]", or use  unique name such as "**myandroid**" or "**Android_nnstreamer**".

File: api/android/README.md [hunk 3]
```diff
@@ -43,43 +43,60 @@ export GSTREAMER_ROOT_ANDROID=$ANDROID_DEV_ROOT/gstreamer-1.0
...
 For example,
 ```bash
-$ firefox  https://developer.android.com/studio
-Then, download "Android Studio" in the /opt folder.
+$ firefox  https://developer.android.com/studio &
+Then, download the **Android Studio** IDE into the /opt folder as follows.
 $ cd /opt
-$ wget https://dl.google.com/dl/android/studio/ide-zips/3.4.0.18/android-studio-ide-183.5452501-linux.tar.gz
-$ tar xvzf ./android-studio-ide-183.5452501-linux.tar.gz
+$ sudo curl -O https://r1---sn-n5hn0ob-pjoe.gvt1.com/edgedl/android/studio/ide-zips/3.6.3.0/android-studio-ide-192.6392135-linux.tar.gz
```
Comment (leemgs) [inline api/android/README.md:62]: Looks good.

File: api/android/README.md (hunk 2 above)
Comment (again4you) [reply api/android/README.md:31]: ACK!

File: api/android/README.md [hunk 4]
```diff
@@ -43,44 +43,53 @@ export GSTREAMER_ROOT_ANDROID=$ANDROID_DEV_ROOT/gstreamer-1.0
 export NNSTREAMER_ROOT=$ANDROID_DEV_ROOT/workspace/nnstreamer
 ```
 
-#### Download Android Studio
+### Install required packages
+Some required packages should be installed as below.
+```bash
+$ sudo apt install subversion
```
Comment (again4you) [inline api/android/README.md:49]: Add `curl`

Comment (myungjoo) [reply api/android/README.md:49]: To whom are you speaking? Please add them :P

Comment (again4you) [reply api/android/README.md:49]: It's for me not to forget it. Because I tested it at home.

**Pull request merged**